CONTRIBUTING.md

```diff
@@ -0,0 +1,724 @@
+# Contributing to BioJulia
+
+:+1::tada: First off, thanks for taking the time to contribute! :tada::+1:
+
+The following is a set of guidelines for contributing to Atom and its packages,
```

Comment:
Why does this say `Atom`?

Reply:
Because I used sevearal other projects as templates to create our Contributing files, one happened to be Atom. I'll fix this.